packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -779,6 +745,39 @@
             </oui-tile>
         </div>
 
+        <!-- SUBSCRIPTIONS -->
+        <div class="col-md-4" data-ng-if="$ctrl.technicalDetails">
```

Comment:
`ng-if` can't be handled in the transclude part ?

Reply:
I'm not really happy with the current code because the "subscription" part is kind of duplicated. I need to display the tile at different places depending on if technicalDetails are available or not.

As you can see here : https://github.com/ovh/manager/pull/3981/files#diff-98049d8b532766dc62f7edce817c34778f8d304c14f5d14d23a952aa8c63465dR725 i had refactored the part but when trying to rebase on develop later i saw that the code changed to billing-subscription-tile. I tried to resolve the conflict without spending too much time on it but i agree that the whole subscription tile should be a single component. If i have the time to refactor it i'll do.

Also i think the ng-if should stay in this template because it's not part of the subscription tile, it's just a conditional to display or not the tile. You would use the tile in another place and the ng-if would be wrong if it was part of the tile.